### ISSUE TITLE
bitusd price oracle

### ISSUE THREAD

**Issue by grctest** [ryanRfox/Stablecoins#2]
bitUSD has 'price oracle' set to 'NONE', but price feeds are provided by the witnesses who could be identified as oracles, no?

**Comment by ryanRfox** [comment]
I agree the block producers are the oracles for SmartCoins. Perhaps you can correct the article to explain BitShares implements a template smart contract that may be configured per asset issued. Go on to explain the finer details thereof.

**Comment by grctest** [comment]
>Lack of a price oracle – In our view one of the most controversial aspects of this design is the absence of any price oracle mechanism, providing the system with real world exchange rates. However any price oracle system is challenging to implement and may introduce several weaknesses and avenues for manipulation. We will talk more about this in part 2. In our view, the only real way around this may be that any stablecoin system may require a price feed from a distributed exchange, which can in theory publish a distributed price feed from real world US Dollar transactions. The distributed exchange in BitShares did not allow “real USD”. A distributed exchange system like Bisq, without a central clearing could in theory allow “real USD” prices and provide a distributed price feed.   Therefore stablecoins may eventually be considered as a layer two technology on top of liquid and robust distributed exchange platforms, should these systems ever emerge.

They also criticize BTS later in the article for a lack of price oracles - not only can committee members & witnesses publish feeds, so can every user in their own private MPA.

**Comment by grctest** [comment]
Further:

> In response to the supposed weakness of a lack of a price peg, Larimer argues in favor of his “hypothesis that the price feed is unnecessary” [ ... ]

MPA have price feeds, fair enough the market price can go above the feed price, but that's a liquidity problem not a price feed problem.
